packages/flutter_tools/lib/src/doctor_validator.dart

```diff
@@ -285,7 +285,7 @@ class NoIdeValidator extends DoctorValidator {
   @override
   Future<ValidationResult> validate() async {
     return ValidationResult(
-      ValidationType.missing,
+      ValidationType.notAvailable,
```

Comment:
it looks like the way this enum is setup now, `notAvailable` is the value we use for "info/warning". However, from reading the code, this is confusing. Can you leave a comment in the code:

```suggestion
      // Info hint to user they do not have a supported IDE installed
      ValidationType.notAvailable,
```